prediction_market_agent_tooling/tools/betting_strategies/kelly_criterion.py

```
@@ -61,7 +62,7 @@ def get_kelly_bet_full(
     estimated_p_yes: float,
     confidence: float,
     max_bet: float,
-    fee: float = 0.0,  # proportion, 0 to 1
+    fees: MarketFees = MarketFees.get_zero_fees(),  # TODO: Remove default value.
```

Ruff

Do not perform function call `MarketFees.get_zero_fees` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)